### PR TITLE
Requesto to merge development branch into master branch

### PR DESCRIPTION
**Pull request by IvanDerlich**
Ready for review for someone from the Microverse staff

### CODE REVIEW

File: assets/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    height: 36px;
+    width: 980px;
+    background-image: linear-gradient(rgb(107,107,107) 50%, rgb(122,122,122) 50%);
+    margin: 0 auto;
+    border-radius: 4px;
+    border-style: solid;
+    border-width: .2px;
+    border-color: rgb(80,80,80);
+    display: flex;
+    flex-direction: horizontal;
```
Comment (azdrenymeri) [inline assets/styles/main.css]: `flex-direction` does not accept a `horizontal` as value

File: assets/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    flex-wrap: wrap;
+    font-family: 'Lucida Grande';
+    color: rgb(110,110,110);
+}
+
+footer ul,
+footer div
+{
+    display: flex;
+    flex-direction: horizontal;
```
Comment (azdrenymeri) [inline assets/styles/main.css]: Same as in line 53 `flex-direction` does not accept a `horizontal` as value

**Review by azdrenymeri — COMMENTED** [review]
## Hello @IvanDerlich  :wave: 
#### Good Job
* The bottom pictures must align with the main header picture as on the original page
screenshot
#### Fix this and add it for a re-review

File: assets/styles/main.css (hunk 1 above)
Comment (IvanDerlich) [reply assets/styles/main.css]: Done

File: assets/styles/main.css (hunk 2 above)
Comment (IvanDerlich) [reply assets/styles/main.css]: Done

**Comment by azdrenymeri** [comment]
### Hello @IvanDerlich  
#### You bottom pictures still are spreading on my widescreen not maintaining its distance on the sides 
screenshot
#### Fix this problem and re add it for a re-review

**Comment by azdrenymeri** [comment]
### It's a :+1: 
#### You can merge now

**Pull request merged**